logbook-core/src/main/java/org/zalando/logbook/BodyFilters.java

```diff
@@ -10,15 +10,15 @@
     }
 
     public static BodyFilter defaultValue() {
-        return accessToken();
+        return replaceProperty("access_token|open_id|id_token", "XXX");
     }
 
-    public static BodyFilter accessToken() {
```

Comment:
Removing this method is a breaking change. You can only add the new one next to it.